quic/s2n-quic-core/src/recovery/bbr/probe_bw.rs

```diff
@@ -92,7 +92,7 @@ pub(crate) struct State {
     /// The current mode for deciding how fast to send
     cycle_phase: CyclePhase,
     /// How the incoming ACK stream relates to our bandwidth probing
-    ack_phase: AckPhase,
+    pub ack_phase: AckPhase,
```

Comment:
It's _probably_ better to have a setter for this so you can add some debug assertions about a state transition. Like we should only transition to a certain phase from another phase.

Reply:
Makes sense, but I think I'm going to end up refactoring this a bit in the next PR and try to see if I can keep this internal to the probe bw state, so I'd prefer to leave it for now. I've added a TODO for now